dotnet/src/Connectors/Connectors.HuggingFace/Core/Models/TextEmbeddingResponse.cs

```diff
@@ -3,7 +3,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Client;
+namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Client.Models;
```

Comment:
Should we keep the folder but reflect in the namespace to minimise the number of usings?